Bundle issue in browser environment with esbuild

### Description

I'm trying to use `hashconnect` package and it uses `hedera-sdk-js` as its main dependency.
When I tried to bundle the project with esbuild, I got nodejs resolutions errors.

I found that if I remove `exports` field in `hedera-sdk-js` package.json it works fine after that.

### Steps to reproduce

https://github.com/fantgeass/issue-example
I've created a simple repo to reproduce an issue.



### Additional context

Esbuild error log when I tried to build.
```
✘ [ERROR] Could not resolve "fs"

    node_modules/@hashgraph/sdk/src/client/NodeClient.js:21:15:
      21 │ import fs from "fs";
         ╵                ~~~~

  The package "fs" wasn't found on the file system but is built into node. Are you trying to bundle
  for node? You can use "platform: 'node'" to do that, which will remove this error.

✘ [ERROR] Could not resolve "util"

    node_modules/@hashgraph/sdk/src/client/NodeClient.js:22:17:
      22 │ import util from "util";...
```

### Hedera network

other

### Version

2.41.0

### Operating system

Linux

--- a/package.json
+++ b/package.json
@@ -16,17 +16,26 @@
     },
     "browser": {
         "./lib/index.js": "./lib/browser.js",
+        "./lib/index.cjs": "./lib/browser.cjs",
         "./lib/network/AddressBookQuery.js": "./lib/network/AddressBookQueryWeb.js",
+        "./lib/network/AddressBookQuery.cjs": "./lib/network/AddressBookQueryWeb.cjs",
         "./lib/encoding/hex.js": "./lib/encoding/hex.browser.js",
+        "./lib/encoding/hex.cjs": "./lib/encoding/hex.browser.cjs",
         "./lib/encoding/utf8.js": "./lib/encoding/utf8.browser.js",
+        "./lib/encoding/utf8.cjs": "./lib/encoding/utf8.browser.cjs",
         "./lib/cryptography/sha384.js": "./lib/cryptography/sha384.browser.js",
+        "./lib/cryptography/sha384.cjs": "./lib/cryptography/sha384.browser.cjs",
         "crypto": false
     },
     "exports": {
         "./package.json": "./package.json",
         ".": {
-            "react-native": "./lib/native.js",
             "types": "./lib/index.d.ts",
+            "react-native": "./lib/native.js",
+            "browser": {
+                "import": "./lib/browser.js",
+                "require": "./lib/browser.cjs"
+            },
             "import": "./lib/index.js",
             "require": "./lib/index.cjs"
         }

